voto/bin/add_sailbuoy.py

```diff
@@ -114,17 +114,29 @@ def add_nrt_sailbuoy(df_in, sb, mission):
     _log.info(f"plotting sailbuoy data from SB{sb} mission {mission}")
     sailbuoy_nrt_plots(ds)
     make_map(ds)
-    if df_in.Leak.any() or df_in.BigLeak.any():
-        leak_alert_email(ds)
+    send_alert_email(ds)
     _log.info(f"Completed add SB{sb} mission {mission}")
 
-
-def leak_alert_email(ds_mission):
-    sb_num = ds_mission.attrs["sailbuoy_serial"]
-    msg = f"Leak detected in Sailbuoy {sb_num}"
-    mailer(msg, leak_mails)
-
-
+def send_alert_email(df_in):
+    sb_num = df_in.attrs["sailbuoy_serial"]
+    msg=[]
+    subset_data = df_in[-15:]
+    if df_in.Leak[-15:].any() or df_in.BigLeak.Leak[-15:].any():
```

Comment:
why df_in.Bigleak.Leak and not just df_in.Bigleak ?

Reply:
Also calling this variable df_in when it is an xarray Dataset is a litle confusing. df suggests pandas DataFrame